conda_smithy/templates/appveyor.yml.tmpl

```diff
@@ -31,7 +29,9 @@ install:
     - cmd: rmdir C:\cygwin /s /q
 
     # Add path, activate `conda` and update conda.
-    - cmd: call %CONDA_INSTALL_LOCN%\Scripts\activate.bat
+    # Around November of 2017, Appveyor changed Miniconda3-x64 from an old 3.4 install to a
+    #     symlink to the current version.
+    - cmd: call C:\Miniconda3-x64\Scripts\activate.bat
```

Comment:
Do we want to always use the Python 3.4 Miniconda? If we want to fix a Python version, why not `36`? It's way more up to date. So should have fewer issues.

Reply:
Also does it matter if 32-bit/64-bit matches in `root` with the platform we are targeting? Guessing not, but wanted to confirm.

Reply:
This is not the Python 3.4 miniconda.  Look at the Appveyor description of the env there: https://www.appveyor.com/docs/build-environment/#miniconda

> Miniconda3 4.3.27 (Python 3.6.2): C:\Miniconda36 or C:\Miniconda3
Miniconda3 4.3.27 x64 (Python 3.6.2): C:\Miniconda36-x64 or C:\Miniconda3-x64

32/64 does not matter.  Conda-build does the right thing with target_platform.

Reply:
Ah ok. Sorry misread.